docker/entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+set -e
+
+cat > "${APP_CONFIG_FILE}" <<EndOfMessage
+---
+keycloak:
```

Comment:
Okay, Spring configuration overriding/merging is a bit weird. Apparently we need to add all keycloak settings here, so also add:
```yaml
  bearer-only: true
  use-resource-role-mappings: true
```

Reply:
Looks like all of the internal configuration is ignored when using externalised config. I don't understand why. https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config

Reply:
Okay, finally found something that works fine:
```
exec java -jar -server \
        "-Djava.awt.headless=true" \
        "-Dmail.mime.decodeparameters=true" \
        "-Djava.security.egd=file:///dev/urandom" \
        "-Dspring.config.location=classpath:/application.yml,${APP_CONFIG_FILE}" \
    "whereabouts-${WHEREABOUTS_VERSION}.jar"


```

Reply:
thanks a lot! I didn't find any other solution yet... Btw, do we need other options for this command, like "-Djava.awt.headless=true" or "-Djava.awt.headless=true"?

Reply:
But also some slight changes are required in the internal config, e.g., the `bearer-only` and `use-resource-role-mappings` need to be moved to `application.yml`.

Reply:
With #12 merged and `"-Dspring.config.location=classpath:/config/application.yml,${APP_CONFIG_FILE}"` added it should work.

Reply:
yes, it works for me, I will push the "Dspring.config" change